docs/rules/no-deprecated.md

```diff
@@ -22,6 +22,8 @@ React.createClass({ /* Class object */ });
 const propTypes = {
   foo: PropTypes.bar,
 };
+
+import {React, PropTypes} from 'react';
```

Comment:
this is `const { React, PropTypes } = require('react');` which is invalid - `React.React` isn't a thing.